_posts/2019-10-03-iw-manchester-2nd-birthday.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Our Manchester office is two!"
+author: david_postle
+---
+
+![Our Manchester office is two!]({{ "/images/IW_2nd_birthday.jpg" }})
+
+# Our Manchester office is two! 
+
+By Dave Postle
+
+Our fast-growing Manchester team celebrated with craft beer, prosecco and a giant cake this week as we reached our second birthday. From our initial launch in the rainy city with just one consultant in a co-working space through to having just taken on our 50th person, the last two years have really flown by. Growth has been rapid as we’ve grown a reputation in Manchester to rival the amazing existing teams in both Leeds and London.
+
+It’s been a fast-paced time as we’ve managed projects large and small, across a wide variety of sectors like transport, healthcare, retail, education, and telecoms - just to name a few. We’ve worked with some amazing companies, some incredibly talented people and developed our technological capabilities infinitely. 
+
+The range of services provided by the local team continues to grow too, extending from the agile, cloud-native engineering that we focused on initially, through digital transformation  including AWS serverless, data engineering, service design and Internet of Things offerings since then. 
```

Comment:
The range of services provided by the local team continues to grow. We've broadened our offering from the agile, cloud-native engineering that we focused on initially; through to digital transformation, AWS serverless, data engineering, service design, and Internet of Things offerings.